content/en/boilerplates/revision-tags-middle.md

```diff
@@ -2,8 +2,8 @@
 ---
 The resulting mapping between revisions, tags, and namespaces is as shown below:
 
-{{< image width="70%"
-link="/docs/setup/upgrade/canary/tags.png"
+{{< image
```

Comment:
I'm not sure I like the 100%, and maybe you can try going back to 70%. Just comparing the same page here vs https://istio.io/latest/docs/setup/upgrade/canary/#stable-revision-labels-experimental for example.

Reply:
> I'm not sure I like the 100%, and maybe you can try going back to 70%. Just comparing the same page here vs https://istio.io/latest/docs/setup/upgrade/canary/#stable-revision-labels-experimental for example.

70% makes it too small.. I have made it 90% which looks okay to me.